src/cartservice/src/cartservice.csproj

```diff
@@ -8,13 +8,13 @@
     <PackageReference Include="Grpc.AspNetCore" Version="2.44.0" />
     <PackageReference Include="Grpc.AspNetCore.HealthChecks" Version="2.46.0" />
     <PackageReference Include="StackExchange.Redis" Version="2.5.43" />
-    <PackageReference Include="OpenTelemetry" Version="1.2.0" />
-    <PackageReference Include="OpenTelemetry.Exporter.OpenTelemetryProtocol" Version="1.2.0" />
-    <PackageReference Include="OpenTelemetry.Extensions.Hosting" Version="1.0.0-rc9.2" />
-    <PackageReference Include="OpenTelemetry.Instrumentation.AspNetCore" Version="1.0.0-rc9.2" />
-    <PackageReference Include="OpenTelemetry.Instrumentation.GrpcNetClient" Version="1.0.0-rc9.2" />
-    <PackageReference Include="OpenTelemetry.Instrumentation.Http" Version="1.0.0-rc9.2" />
-    <PackageReference Include="OpenTelemetry.Instrumentation.StackExchangeRedis" Version="1.0.0-rc9.2" />
+    <PackageReference Include="OpenTelemetry" Version="1.3.0" />
```

Comment:
you can remove this package, as it is brought by the exporter.

Reply:
Thx @cijothomas!
Updated!